posts/championship_forecasting.md

```diff
@@ -0,0 +1,285 @@
+---
+title: "Forecasting Championship Results using Time Series and Nixtla"
+description: "Learn how to forecast championship standings using Nixtla's StatsForecast library."
+image: "/images/title_image.svg"
+categories: ["TimeGPT Forecasting"]
+tags:
+  - StatsForecast
+  - forecasting
+  - championship
+  - sports analytics
+  - time series
+  - AutoARIMA
+author_name: Piero Paialunga
+author_image: "/images/authors/piero.jpg"
+author_position: Data Scientist
+publication_date: 2025-01-12
```

Comment:
Can you change this to the latest day? It doesn't need to be exact; you can use today.